Makefile

```diff
@@ -8,37 +8,23 @@ BUILD_FLAGS := $(if $(BUILD_FLAGS),$(BUILD_FLAGS),--load --no-cache)
 BUILDER_NAME = k8s-wait-for-builder
```

Comment:
may want to ditch the makefile changes overall, was just trying to make `make test` work locally based on the dcasavant repo

Reply:
I'm inclined to say that we ditch your changes at the moment.